220-ofcom-22000.md

```diff
@@ -1,4 +1,58 @@
 # 2.20. OFCOM \(-22000\)
+## ofcom\_odbc\: SQLConnect failed
```

Comment:
For the error code title, please use the one which is printed out when you do oferror.
For example, use OFCOM_ERROR_ODBC (-22110).

```
[oframe7@oframe7 ~]$ oferror 22110
/*
/* error no  : -22110
/* error code: OFCOM_ERR_ODBC
/* error msg : ODBC function failed.
/*
```